kode

```diff
@@ -485,8 +485,7 @@ tasks:
         -e "KUBE_DNS_SERVICE_HOST=${KUBE_DNS_SERVICE_HOST}" \
         -e "KUBE_DNS_SERVICE_PORT=${KUBE_DNS_SERVICE_PORT}" \
         --rm -w "${wd}" \
-        "${image}" bash -c "${cmd}"
-        # | cloudwatch-logger -t "${log_group}" ${DEPLOYMENT_ID/stdout}
+        "${image}" bash -c "${cmd}" | cloudwatch-logger -t "${log_group}" ${DEPLOYMENT_ID/stdout}
```

Comment:
I think `${DEPLOYMENT_ID/stdout}` should be `${DEPLOYMENT_ID}/stdout` and `${deploy_id}` in line 615 to be `${deploy_id}/stdout`, but LGTM overall!

Reply:
Ah, so bash evaluates `${DEPLOYMENT_ID/stdout}` to just `${DEPLOYMENT_ID}`, so this should work anyway, for stdout only. LGTM.

Reply:
Thanks for checking about bash evaluations 👍